README.md

```diff
@@ -1414,18 +1418,16 @@ extension DataRequest {
             guard error == nil else { return .failure(BackendError.network(error: error!)) }
 
             let jsonResponseSerializer = DataRequest.jsonResponseSerializer(options: .allowFragments)
-            let result = jsonResponseSerializer.serializeResponse(request, response, data, error)
+            let result = jsonResponseSerializer.serializeResponse(request, response, data, nil)
```

Comment:
Added in [0a05e2b](https://github.com/Alamofire/Alamofire/commit/880dc772a12d1ec490eb8aee029115ec8).